BookWorm/BookWorm/View Controllers/SearchViewController.m

```diff
@@ -76,7 +77,7 @@ - (IBAction)clickedSearch:(id)sender {
            }
            else {
                NSDictionary *dataDictionary = [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingMutableContainers error:nil];
-               // NSLog(@"%@", dataDictionary);
+               NSLog(@"%@", dataDictionary);
```

Comment:
nit: either remove the random debugging logging, or give a bit more context instead of just printing out the raw data (with only raw data, not useful to read through in the console)